### ISSUE TITLE
map CallSignature

### ISSUE THREAD

**Issue by ctaggart** [fable-compiler/ts2fable#23]
From #19.

**Comment by alfonsogarciacaro** [comment]
For the call signature, I think ts2fable already uses `Invoke` with an `Emit` attribute (although we could use an specialized attribute):

```typescript
class Foo {
   (source: string, subString: string): boolean;
}
```

```fsharp
type Foo =
    [<Emit("$0($1...)">]
    abstract Invoke: source:string*subString:string -> bool
```

**Comment by ctaggart** [comment]
Fixed by #33, I think. @alfonsogarciacaro, can you review the resulting code changes to typescript.fs in #33?
